README.md

```diff
@@ -114,23 +114,15 @@ To use the GitHub Action with Codacy integration:
 
     This setting enables Codacy to wait for the results of the local analysis before resuming the analysis of your commits.
 
-2.  On Codacy, obtain one of the following tokens:
+2.  Set up an API token to allow the GitHub Action to authenticate on Codacy:
 
-    -   [Project API Token](https://docs.codacy.com/codacy-api/api-tokens/#project-api-tokens)
+    -   **If you're setting up coverage for one repository**, [obtain a project API token](../codacy-api/api-tokens/#project-api-tokens) and store it as an [encrypted secret for your **repository**](https://docs.github.com/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository) with the name `CODACY_PROJECT_TOKEN`.
 
-        You need the Project API Token to allow the Codacy Analysis CLI to authenticate to Codacy when reporting the analysis results.
+    -   **If you're setting up and automating coverage for multiple repositories**, [obtain an account API token](../codacy-api/api-tokens/#account-api-tokens) and store it as an [encrypted secret for your **organization**](https://docs.github.com/en/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-an-organization) with the name `CODACY_API_TOKEN`.
```

Comment:
What do you think about instructing the users to store the account API token at the organization level? Like this, they would need to set up the token only once.